### PR TITLE
Kubectl base image

### PR DESCRIPTION
**Pull request by taimir**
First attempt at creating a Docker container which will contain `kubectl` and the addons yaml files.

@batikanu check it out

### CODE REVIEW

File: addons/run.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+echo "Addon creation started"
+WORK_DIR="/opt/kube-addons"
+
+# create heapster addon
+if [ "$ENABLE_CLUSTER_MONITORING" == "monasca" ]; then
+    # TODO (Atanas): sed monasca configuration and start monasca heapster
+    echo "Monasca should start"
+else
+    ${WORK_DIR}/kubectl create -f ${WORK_DIR}/heapster/standalone/heapster-service.yaml
```
Comment (batikanu) [inline addons/run.sh]: ${WORK_DIR}/kubectl create -f ${WORK_DIR}/heapster/standalone/

**Comment by batikanu** [comment]
LGTM

File: addons/run.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+echo "Addon creation started"
+WORK_DIR="/opt/kube-addons"
+cd ${WORK_DIR}
+
+# create heapster addon
+if [ "$ENABLE_CLUSTER_MONITORING" == "monasca" ]; then
+    # TODO (Atanas): sed monasca configuration and start monasca heapster
+    echo "Monasca should start"
+else
+    ./kubectl create -f heapster/standalone/
```
Comment (zreigz) [inline addons/run.sh:11]: How kubectl knows where is API server ?

Comment (cheld) [inline addons/run.sh:11]: localhost with default port. It is running on master

**Pull request merged**